my_project/tab_outdoor_comfort/app_outdoor_comfort.py

```diff
@@ -213,7 +213,7 @@ def update_tab_utci_category(ts, var, global_local, df, meta, si_ip):
         title="Thermal stress",
         titleside="top",
         tickmode="array",
-        tickvals=[4, 3, 2, 1, 0, -1, -2, -3, -4, -5],
+        tickvals=[4.75, 3.9, 2.8, 1.65, 0.55, -0.55, -1.65, -2.75, -3.9, -4.75],
```

Comment:
great work, but could you replace this static definition with np.linspace(start, stop, number) so the code is easier to read and maintain something like np.linspace(-4.75, 4.75, 10) I have not tested this.